### PR TITLE
feat: remove moderation options from research items

### LINKED ISSUE #2571: [change] remove moderation options from draft research items
**Is your feature request related to a problem? Please describe.**
Moderators are given the option to approve / reject a draft research item.  This could lead to them approving an item that is not finished.

**Describe the solution you'd like**
Remove the moderation section from the research article view.

**Describe alternatives you've considered**
N/A

**Additional context**

### PR DESCRIPTION
**Pull request by bagiyal**
PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description
This pull request introduces a new feature to the project that removes moderation options from research items. The changes do not introduce any breaking changes and do not affect existing functionality. This enhancement aims to improve the user experience by simplifying the process of managing research items.

## Closes

#2571

### CODE REVIEW

File: src/pages/Research/Content/ResearchDescription.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ const ResearchDescription = ({ research, isEditable, ...props }: IProps) => {
             </AuthWrapper>
           ) : null}
           {/* Check if research should be moderated */}
-          {props.needsModeration && (
+          {props.needsModeration && research.moderation !== 'draft' && (
```
Comment (AlfonsoGhislieri) [inline src/pages/Research/Content/ResearchDescription.tsx:123]:
```suggestion
          {props.needsModeration && research.moderation === 'awaiting-moderation' && (
```

This is the logic we would want

**Comment by evakill** [comment]
thanks for the update @bagiyal! 
the test failure is due to the format of your last commit message. to keep things tidy, we like to use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) and enforce them in our lint rules. to fix the commit, i would suggest using `git commit --amend`, revising the commit message to use the conventional commit format, then pushing with `git push --force`

**Comment by AlfonsoGhislieri** [comment]
Thanks for the update and amending the commit. Still tests not passing due to linting issues in the file, you need to make sure you are running prettier to auto format the code ! https://prettier.io/

**Review by AlfonsoGhislieri — APPROVED** [review]

**Comment by AlfonsoGhislieri** [comment]
@all-contributors add @bagiyal for code

**Pull request merged**